<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>摄像头参数</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>摄像头设备</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboCamera"/>
          </item>
          <item>
           <widget class="QCheckBox" name="checkStillImage">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>支持抓图</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkVideo">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>支持录像</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>录像时静音</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>视频保存文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>录制时间：4秒</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCameraViewfinder" name="viewFinder" native="true"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openCamera"/>
   <addaction name="closeCamera"/>
   <addaction name="capture"/>
   <addaction name="startRecorder"/>
   <addaction name="stopRecorder"/>
   <addaction name="exit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openCamera">
   <property name="icon">
    <iconset theme="开启摄像头" resource="resources.qrc">
     <normaloff>:/images/camera.png</normaloff>:/images/camera.png</iconset>
   </property>
   <property name="text">
    <string>开启摄像头</string>
   </property>
   <property name="toolTip">
    <string>开启摄像头</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="closeCamera">
   <property name="icon">
    <iconset>
     <normalon>:/images/close.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>关闭摄像头</string>
   </property>
   <property name="toolTip">
    <string>关闭摄像头</string>
   </property>
  </action>
  <action name="capture">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/capture.png</normaloff>:/images/capture.png</iconset>
   </property>
   <property name="text">
    <string>抓图</string>
   </property>
   <property name="toolTip">
    <string>抓图</string>
   </property>
  </action>
  <action name="startRecorder">
   <property name="icon">
    <iconset>
     <normalon>:/images/recorder.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>开始录像</string>
   </property>
   <property name="toolTip">
    <string>开始录像</string>
   </property>
  </action>
  <action name="stopRecorder">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>停止录像</string>
   </property>
   <property name="toolTip">
    <string>停止录像</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header location="global">qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
